lib/libvfio-user.c

```diff
@@ -236,13 +232,15 @@ region_access(vfu_ctx_t *vfu_ctx, size_t region, char *buf,
     }
 
 out:
-    if (ret != (ssize_t)count) {
+    if (unlikely(ret != (ssize_t)count)) {
         vfu_log(vfu_ctx, LOG_DEBUG, "region%zu: %s (%#llx:%zu) failed: %m",
                 region, verb, (ull_t)offset, count);
-    } else {
+    }
+#ifdef DEBUG
```

Comment:
instead can you define:

#else
static void debug_region_access(..)
{}
#endif

when non-debug?

Reply:
done